app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  # Association
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
Kindly add the foreign key relationship between the User and comments/likes😉 as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)👍🏼.

```suggestion
  has_many :comments, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id
```

Reply:
@Ibizugbe thank you for the feedback